recipe/conda_build_config.yaml

```diff
@@ -602,6 +602,8 @@ tk:
   - 8.6                # [not ppc64le]
 tiledb:
   - 1.7
+uhd:
+  - 3.15.0
```

Comment:
can it just be `3.15`?

Reply:
The `run_exports` pin is `x.x.x`, which matches the SOVERSION on the uhd library. So I think that means the third part is necessary, unless `3.15` would imply the `.0` in this case?

Reply:
yep OK